.github/workflows/go-test-config.json

```diff
@@ -1,3 +1,7 @@
 {
-  "skip32bit": true
+  "skip32bit": true,
+  "skipOSes": [
+    "windows",
+    "macos"
```

Comment:
is there a reason it doesn't work on mac?

Reply:
It does work on both mac and windows. It just needs installation of c bindings. I chose to postpone setting this up for CI in favour of testing this in dev to see if it fits the task in hand. _if_ it does then I intend to attend this debt.

Reply:
https://github.com/ipni/dhstore/issues/48